api/app/signals/apps/api/v1/serializers/signal.py

```diff
@@ -332,46 +332,35 @@ class PublicSignalCreateSerializer(serializers.ModelSerializer):
     class Meta(object):
         model = Signal
         fields = (
-            'id',
-            'signal_id',
             'source',
             'text',
             'text_extra',
             'location',
             'category',
             'reporter',
-            'status',
-            'priority',
-            'type',
-            'created_at',
-            'updated_at',
             'incident_date_start',
             'incident_date_end',
-            'image',
-            'attachments',
             'extra_properties',
         )
-        read_only_fields = (
```

Comment:
Same thing `signal_id` should be read-only

Reply:
see above

Reply:
see above